dp/buy_sell_stock.py

```diff
@@ -23,12 +23,13 @@
 def max_profit(prices):
     """
     input: [7, 1, 5, 3, 6, 4]
-    diff : [X,-6, 5,-2, 3,-2]
+    diff : [X,-6, 4,-2, 3,-2]
```

Comment:
Thanks!
But what does 'nit' mean?
And I used space around ',', kept same format with original lines.